nubis/puppet/files/confd/templates/traefik.toml.tmpl

```diff
@@ -224,6 +224,9 @@ sans = [
 [[acme.domains]]
 main = "planet-bugzilla.allizom.org"
 
+[[acme.domains]]
```

Comment:
This is in the stage configuration block, instead of stage.

Reply:
Thanks. I moved it to the prod block.